src/components/Todo-List.vue

```diff
@@ -1,5 +1,6 @@
 <template>
   <div class="todo-list">
+    <todo-add @todoAdded="addTodo"></todo-add>
     <ul>
       <li v-for="todo in todos" class="todo-item">
```

Comment:
I'm looking for warnings about a key not present but I don't see any. I've tried FF and Chrome on Mac so far.